velox/docs/configs.rst

```diff
@@ -220,7 +220,43 @@ to an existing partition.
 True if appending data to an existing unpartitioned table is allowed.
 Currently this configuration does not support appending to existing partitions.
 
-
+``Amazon S3 Configuration``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :widths: 35 10 40
+   :header-rows: 1
+
+   * - Property Name
+     - Type
+     - Description
+   * - hive.s3.use-instance-credentials
+     - bool
+     - Use the EC2 metadata service to retrieve API credentials (defaults to true). This works with IAM roles in EC2.
+   * - hive.s3.aws-access-key
+     - string
+     - Default AWS access key to use.
+   * - hive.s3.aws-secret-key
```

Comment:
Duplicate?

Reply:
Not a duplicate. The above one is `hive.s3.aws-access-key` (access vs. secret)